contrib/native/client/src/test/CMakeLists.txt

```diff
@@ -29,6 +30,7 @@ include_directories(${CPPUNIT_INCLUDE_DIR})
 include_directories(${Boost_INCLUDE_DIRS})
 include_directories(${PROTOBUF_INCLUDE_DIR})
 include_directories(${Zookeeper_INCLUDE_DIRS})
+include_directories(${OPENSSL_INCLUDE_DIR})
```

Comment:
@vvysotskyi [cc. @agozhiy ] - I have added this extra include directory to include open SSL header files otherwise DrillClientTest will show compilation issues.